frontend/src/scenes/notebooks/Nodes/NotebookNodeBacklink.tsx

```diff
@@ -0,0 +1,92 @@
+import { mergeAttributes, Node, NodeViewProps } from '@tiptap/core'
+import { NodeViewWrapper, ReactNodeViewRenderer } from '@tiptap/react'
+import { InsightModel, NotebookNodeType } from '~/types'
+import { Link } from '@posthog/lemon-ui'
+import { IconGauge, IconBarChart, IconFlag, IconExperiment, IconLive, IconPerson, IconCohort } from 'lib/lemon-ui/icons'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { urls } from 'scenes/urls'
+import { useActions, useValues } from 'kea'
+import { notebookSidebarLogic } from '../Notebook/notebookSidebarLogic'
+import { notebookLogic } from '../Notebook/notebookLogic'
+import clsx from 'clsx'
+
+const ICON_MAP = {
+    dashboards: <IconGauge />,
+    insights: <IconBarChart />,
+    feature_flags: <IconFlag />,
+    experiments: <IconExperiment />,
+    events: <IconLive width="1em" height="1em" />,
+    persons: <IconPerson />,
+    cohorts: <IconCohort />,
+}
+
+const Component = (props: NodeViewProps): JSX.Element => {
+    const { shortId } = useValues(notebookLogic)
+    const { notebookLinkClicked } = useActions(notebookSidebarLogic)
+
+    const type: TaxonomicFilterGroupType = props.node.attrs.type
+    const href: string | undefined = backlinkHref(props.node.attrs.id, type)
+    const title: string = props.node.attrs.title
+
+    const isViewing = window.location.pathname === href
```

Comment:
May or may not be overkill but you can use the `router` logic to get the location info which I believe reacts to changes. That way if they navigate away, the component should get updated.